pkg/cloudprovider/providers/fake/fake.go

```diff
@@ -26,7 +26,7 @@ import (
 
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
-	cloudprovider "k8s.io/cloud-provider"
```

Comment:
This would result in compile error below with `cloudprovider.ErrNotImplemented`.

Reply:
wow, TIL this still compiles. I prefer we keep it still for consistency, what do you think? :)